import-service/src/libs/validator.ts

```diff
@@ -0,0 +1,15 @@
+const PNG_OR_JPG_REGEX = /^[a-zA-Z0-9]{3,}\.(jpg|png)$/i;
+const CSV_FILE_NAME_REGEX = /^[a-zA-Z0-9]+\.csv$/i;
+const GUID_REGEX = /^[0-9A-Fa-f\-]{36}$/;
+
+const isImageFileName = (n: string) => PNG_OR_JPG_REGEX.test(n);
+const hasProperty = <T>(obj: T, name: string) => obj.hasOwnProperty(name);
+const isGuid = (v: string) => GUID_REGEX.test(v);
+const isCSVFileName = (n: string) => CSV_FILE_NAME_REGEX.test(n);

```

Comment:
some csv names cannot be passed by this regex. e.g. I had a couple of such files, they were not passed. `1,2,3.csv` and `test - Copy.csv`

Reply:
an eagle eye)) thank you) I fixed and deployed it, now specified variants work as well

Reply:
you could try to repeat your upload, it works